### ISSUE TITLE
Conflict with WooCommerce 3.2.3

### ISSUE THREAD

**Issue by jasonbahl** [wp-graphql/wp-graphql#296]
There is some sort of conflict between WPGraphQL v0.0.21 and WooCommerce 3.2.3. 

Initially reported here: #295 

I'm not sure what the issue is yet but I've confirmed that when those are the only 2 plugins active, the response from a GraphQL query is: 

```
{
  "errors": [
    {
      "message": "Internal server error",
      "category": "internal"
    }
  ]
}
```

**Comment by jasonbahl** [comment]
@cr101 I discovered what the conflict is. I have a fix coming shortly.

**Comment by jasonbahl** [comment]
@cr101 I opened this PR to address this #297 

Planning to get it out today. Thanks!

**Comment by jasonbahl** [comment]
Addressed by #297
